ReactNativeTraining/src/redux/appNavigator/actions.js

```diff
@@ -0,0 +1,9 @@
+import { groups, contacts, messages } from '../../services/services';
+
+const loadData = id => dispatch => {
+  messages().then(response => dispatch({ type: 'LOAD_MESSAGES', payload: response.data }));
+  groups(id).then(response => dispatch({ type: 'LOAD_GROUPS', payload: response.data }));
+  contacts().then(response => dispatch({ type: 'LOAD_CONTACTS', payload: response.data }));
```

Comment:
Great! Just for you to keep this in mind, what if api doesn't respond? You need to dispatch an action that inform the error, and save the error in state if u want to use for something. But for the purpose of the training, this is fine.